spatialmath/geo_obstacle.go

```diff
@@ -127,6 +129,7 @@ func GetCartesianDistance(p, q *geo.Point) (float64, float64) {
 // Because the function we use to project a point on a spheroid to a plane is nonlinear, we linearize it about a specified origin point.
 func GeoPointToPose(point, origin *geo.Point) Pose {
 	latDist, lngDist := GetCartesianDistance(origin, point)
+	// NOTE: I don't understand why Bearing is a float from -180 to +180 & not degrees i.e. 0 - 360
```

Comment:
That's how the SOUP we use implements it, ultimately a raw `Atan2` call: https://github.com/kellydunn/golang-geo/blob/2c6d5d781da2b42e60f1d7507c1055ef625d8652/point.go#L89